doc/frameworks/djl/using_djl.rst

```diff
@@ -19,238 +16,53 @@ For general information about using the SageMaker Python SDK, see :ref:`overview
+Alternatively, you can provide full specifications to the DJLModel to have full control over the model configuration:
 
 .. code:: python
 
-    # Create a model using the DeepSpeed backend
-    deepspeed_model = DeepSpeedModel(
-        "s3://my_bucket/my_saved_model_artifacts/", # This can also be a HuggingFace Hub model id
-        "my_sagemaker_role",
-        dtype="bf16",
-        task="text-generation",
-        tensor_parallel_degree=2, # number of gpus to partition the model across using tensor parallelism
-    )
-
-    # Create a model using the HuggingFace Accelerate backend
-
-    hf_accelerate_model = HuggingFaceAccelerateModel(
-        "s3://my_bucket/my_saved_model_artifacts/", # This can also be a HuggingFace Hub model id
-        "my_sagemaker_role",
-        dtype="fp16",
-        task="text-generation",
-        number_of_partitions=2, # number of gpus to partition the model across
-    )
-
-    # Create a model using the FasterTransformer backend
-
-    fastertransformer_model = FasterTransformerModel(
-        "s3://my_bucket/my_saved_model_artifacts/", # This can also be a HuggingFace Hub model id
-        "my_sagemaker_role",
-        data_type="fp16",
+    djl_model = DJLModel(
+        model_id="<hf hub id | s3 uri>",
+        role="my_sagemaker_role",
         task="text-generation",
-        tensor_parallel_degree=2, # number of gpus to partition the model across
+        engine="Python",
+        env={
+            "OPTION_ROLLING_BATCH": "lmi-dist",
+            "TENSOR_PARALLEL_DEGREE": "2",
+            "OPTION_DTYPE": "bf16",
+            "OPTION_MAX_ROLLING_BATCH_SIZE": "64",
+        },
+        image_uri=<djl lmi image uri>,
     )
 
-    # Deploy the model to an Amazon SageMaker Endpoint and get a Predictor
-    deepspeed_predictor = deepspeed_model.deploy("ml.g5.12xlarge",
-                                                 initial_instance_count=1)
-    hf_accelerate_predictor = hf_accelerate_model.deploy("ml.g5.12xlarge",
-                                                         initial_instance_count=1)
-    fastertransformer_predictor = fastertransformer_model.deploy("ml.g5.12xlarge",
-                                                                 initial_instance_count=1)
-
-Regardless of which way you choose to create your model, a ``Predictor`` object is returned. You can use this ``Predictor``
-to do inference on the endpoint hosting your DJLModel.
-
 Each ``Predictor`` provides a ``predict`` method, which can do inference with json data, numpy arrays, or Python lists.
 Inference data are serialized and sent to the DJL Serving model server by an ``InvokeEndpoint`` SageMaker operation. The
 ``predict`` method returns the result of inference against your model.
 
 By default, the inference data is serialized to a json string, and the inference result is a Python dictionary.
 
-Model Directory Structure
```

Comment:
is all of this removed information being moved to the LMI docs on the AWS documentation site?

Reply:
Yes, we have this all on our LMI docs site.

We are not publishing on AWS docs anymore, but have aligned on all our docs living here https://docs.djl.ai/docs/serving/serving/docs/lmi/index.html. The model directory structure specifically is here https://docs.djl.ai/docs/serving/serving/docs/lmi/deployment_guide/model-artifacts.html